src/Compilers/CSharp/Portable/DocumentationComments/DocumentationCommentIDVisitor.PartVisitor.cs

```diff
@@ -146,8 +146,14 @@ public override object VisitTypeParameter(TypeParameterSymbol symbol, StringBuil
 
                 // Is this a type parameter on a type?
                 Symbol containingSymbol = symbol.ContainingSymbol;
-                if (containingSymbol.Kind == SymbolKind.NamedType)
+                if (containingSymbol.Kind == SymbolKind.Method)
                 {
+                    builder.Append("``");
+                }
+                else 
+                {
+                    Debug.Assert(containingSymbol is NamedTypeSymbol);
```

Comment:
No there is not and I couldn't come up with one, none of the Watson dumps include heap. The code in the block should work fine for ErrorTypeSymbols, in fact it doesn't even depend on the type or kind of this symbol.